requirements.txt

```diff
@@ -4,9 +4,9 @@
 #
 #    pip-compile --generate-hashes --output-file=requirements.txt requirements.in
 #
---only-binary :all:
 --no-binary grapheme
 --no-binary grpclib
+--only-binary :all:
```

Comment:
Apparently, the order of the options matters (see also #121), and this change made grapheme fail to install.

Reply:
Yep, I don't know if dependabot uses pip-tools, but at least the pip-tools fix at https://github.com/python/release-tools/pull/130 hasn't been released yet. I'll give them a ping.

This also wasn't detected because these PRs don't trigger the workflow, let's change that to prevent this happening again.

Reply:
Please see PR https://github.com/python/release-tools/pull/134.